src/app/component/layout/components/search-popup/search-popup.component.html

```diff
@@ -16,51 +16,43 @@
   <div class="fixed-wrapper">
     <div class="adaptive-center-wrapper">
       <div class="search-content-wrapper">
-        <div class="remaining-items" *ngIf="inputValue && itemsFound">
+        <div class="remaining-items" *ngIf="inputValue">
           {{ itemsFound }} items found
         </div>
         <ng-template [ngIf]="isNewsSearchFound || isTipsSearchFound">
           <div class="content-news" *ngIf="isNewsSearchFound">
-            <a [routerLink]="['/news']"
-               (click)="closeSearch()">
+            <a [routerLink]="['/news']" (click)="closeSearch()">
               <h3>News</h3>
             </a>
             <div class="flex-container">
-              <app-search-item *ngFor="let element of newsElements"
-                               [searchModel]="element"
-                               (closeSearch)="closeSearch()"></app-search-item>
+              <app-search-item *ngFor="let element of newsElements" [searchModel]="element"
+                (closeSearch)="closeSearch()"></app-search-item>
             </div>
           </div>
 
           <div class="content-tips" *ngIf="isTipsSearchFound">
-            <a [routerLink]="['/welcome']"
-               pageScroll
-               href="#tips"
-               [pageScrollDuration]="500"
-               (click)="closeSearch()">
+            <a [routerLink]="['/welcome']" pageScroll href="#tips" [pageScrollDuration]="500" (click)="closeSearch()">
               <h3>Tips & Tricks</h3>
             </a>
             <div class="flex-container">
-              <app-search-item *ngFor="let element of tipsElements"
-                               [searchModel]="element"
-                               (closeSearch)="closeSearch()"></app-search-item>
+              <app-search-item *ngFor="let element of tipsElements" [searchModel]="element"
+                (closeSearch)="closeSearch()"></app-search-item>
             </div>
           </div>
 
-          <a [routerLink]="['/search']"
-             (click)="closeSearch()"
-             class="all-results-link">
+          <a [routerLink]="['/search']" (click)="closeSearch()" class="all-results-link">
             See all results
           </a>
         </ng-template>
 
         <ng-template [ngIf]="(isNewsSearchFound === false && isTipsSearchFound === false)">
```

Comment:
unnecessary `ng-template`

Reply:
i removed ng-template, check

Reply:
Not everywhere and very sloppy. There is still `ng-template` wraps all component:
```html
<ng-template [ngIf]="isSearchClicked">
  <div class="search-bar-wrapper">
    <div class="center-flex-wrapper">
      <div class="search-icon"></div>
```
There should be used `ng-container` and normal `*ngIf` instead of `[ngIf]`.
Next one, you changed `ng-template` but added unnecessary `div`, like here:
```html
<div *ngIf="(isNewsSearchFound === false && isTipsSearchFound === false)">
          <app-search-not-found 
            [inputValue]="inputValue" 
            (closeUnsuccessfulSearchResults)="closeSearch()">
          </app-search-not-found>
        </div>
```
Here is also unnecessary `div`:
```html
<div *ngIf="isNewsSearchFound || isTipsSearchFound">
```
because contains only 2 blocks with appropriate conditions, according them will be show first or second block despite the condition in mentioned `div`
Here is also bad
```js
isNewsSearchFound === false
```